.drone.yml

```diff
@@ -75,6 +75,16 @@ steps:
     event:
     - push
 
+- name: secrets leaks test
+  pull: default
+  image: python:3.7.3-alpine3.8
+  commands:
+  - apk --update add bash git
+  - pip install -r requirements.txt
+  - trufflehog --exclude_paths trufflehog-exclude.txt --regex .
```

Comment:
trufflehog-exclude.txt is not present, probably it is ok?

Reply:
present in root folder